cnames_active.js

```diff
@@ -111,6 +111,7 @@ var cnames_active = {
   "agrawalnaman": "agrawalnaman.github.io",
   "agrawalrohit": "rohit0803.github.io/agrawal.github.io",
   "agros": "agrosjs.github.io",
+  "agrume": "cname.vercel-dns.com",
```

Comment:
```suggestion
  "agrume": "cname.vercel-dns.com", // noCF
```
As the CI error notes, you'll want the Cloudflare proxy disabled for Vercel